<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphWindow</class>
 <widget class="QMainWindow" name="graphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="chartView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="startTime">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2016</year>
      <month>11</month>
      <day>15</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endTime">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2016</year>
      <month>11</month>
      <day>15</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2016</year>
      <month>11</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>StartTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EndTime</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
